platform-includes/sourcemaps/upload/primer/javascript.cloudflare.mdx

```diff
@@ -7,6 +7,6 @@ If you can't find the tool of your choice in the list below, we recommend you ch
 
 </Alert>
 
-<Include name="sourcemaps/overview/javascript.cloudflare.mdx" />
```

Comment:
This include was simply being ignored as it could not be found. With the new system, this causes a build error so I fixed it.

Check [current](https://docs.sentry.io/platforms/javascript/guides/cloudflare/sourcemaps/uploading/) vs [fixed](https://sentry-docs-git-byk-cimdx-cache.sentry.dev/platforms/javascript/guides/cloudflare/sourcemaps/uploading/)